src/snek5000_cbox/solver.py

```diff
@@ -91,5 +122,51 @@ def create_default_params(cls):
 
         return params
 
+    def __init__(self, params):
+
+        if params.oper.delta_T_lateral == 1.0 and params.oper.delta_T_vertical == 0.0:
```

Comment:
Could you please add short tests to cover all these possibilities?

Reply:
Done!